Source/WebCore/platform/graphics/iso/ISOTrackEncryptionBox.cpp

```diff
@@ -87,7 +87,7 @@ bool ISOTrackEncryptionBox::parsePayload(DataView& view, unsigned& offset)
 
     if (m_defaultIsProtected == 1 && !m_defaultPerSampleIVSize) {
         int8_t defaultConstantIVSize = 0;
-        if (!checkedRead<int8_t>(defaultConstantIVSize, view, offset, BigEndian))
+        if (!checkedRead<int8_t>(defaultConstantIVSize, view, offset, BigEndian) || defaultConstantIVSize <= 0)
```

Comment:
Seems like this unnecessarily changes behavior when `defaultConstantIVSize` is 0. Should we do this instead?
```suggestion
        if (!checkedRead<int8_t>(defaultConstantIVSize, view, offset, BigEndian) || defaultConstantIVSize < 0)
```